chainer/links/connection/n_step_lstm.py

```diff
@@ -88,6 +88,37 @@ class NStepLSTM(NStepLSTMBase):
 
     .. seealso::
         :func:`chainer.functions.n_step_lstm`
+    
+    .. admonition:: Example
+
+        >>> '''Read forward() method below first'''
+        >>> import chainer
+        >>> from chainer import Variable
+        >>> from chainer import links as L
+        >>> import numpy as np
+        >>> chainer.global_config.dtype = np.float64  # setting default tenspr dtype.
+        >>> dropout_ratio = 0.0
+        >>> in_size , seq_len, n_layers, out_size = 2, 4, 2, 3
+        >>> batch = 5
+        >>> xs = [Variable(np.random.rand(seq_len, in_size)) for i in range(batch)]
+        >>> [x.shape for x in xs]
+        [(4, 2), (4, 2), (4, 2), (4, 2), (4, 2)]
+        >>> lstm = L.NStepLSTM(n_layers, in_size, out_size, dropout_ratio)
+        >>> hy, cy, ys = lstm(None, None, xs) # passing no hidden or cell state
```

Comment:
The style requires two space before `#` that starts a comment
```suggestion
        >>> hy, cy, ys = lstm(None, None, xs)  # passing no hidden or cell state
```